packaging/flavorgen/flavors/crs/cpi.go

```diff
@@ -45,11 +43,6 @@ func addCPILabels(o metav1.Object, cpiInfraLabelValue string) {
 // CreateCrsResourceObjectsCPI creates the api objects necessary for CSI to function.
 // Also appends the resources to the CRS.
 func CreateCrsResourceObjectsCPI(crs *addonsv1.ClusterResourceSet) []runtime.Object {
-	serviceAccount := cloudprovider.CloudControllerManagerServiceAccount()
-	addCPILabels(serviceAccount, "service-account")
-	serviceAccountSecret := newSecret(serviceAccount.Name, serviceAccount)
-	appendSecretToCrsResource(crs, serviceAccountSecret)
-
 	credentials := map[string]string{}
```

Comment:
I assume all the things we drop here are part of cpi.yaml? (or something which provides whatever we need instead)

Reply:
Yes.